MIGRATION-V6.md

```diff
@@ -0,0 +1,146 @@
+- Interface `ListVoiceModelConstants` renamed to `ListCustomModelConstants`
+- Interface `VoiceModel` renamed to `CustomModel`
+- Use of `VoiceModel[]` replaced with interface `CustomModels`
+
+#### Visual Recognition v3
+- Interface `Class` property `_class` renamed `class`
+- Interface `ClassResult` property `_class` renamed `class`
+
+#### Visual Recognition v4
+- Interface `Collection` property `training_status` type changed from `TrainingStatus` to `CollectionTrainingStatus`
+- Interface `ObjectDetail` property `location` type changed from `Location` to `ObjectDetailLocation`
+
+
+### New Features by Service
+
+#### Assistant v1
+- `includeCount` parameter added to several methods: Whether to include information about the number of records that satisfy the request, regardless of the page limit.
+- `bulkClassify()` method added: Identify intents and entities in multiple user utterances.
+- Interface `DialogNodeContext` added
+- Interface `DialogNodeOutput` property `integrations` added: Output intended for specific integrations. For more information, see the [documentation](https://cloud.ibm.com/docs/assistant?topic=assistant-dialog-responses-json)
+
+#### Assistant v2
+- `bulkClassify()` method added: Identify intents and entities in multiple user utterances.
+
+#### Compare Comply
+- Interface `Category` property `modification` added: The type of modification of the feedback entry in the updated labels response.
+- Interface `TypeLabel` property `modification` added: The type of modification of the feedback entry in the updated labels response.
+
+#### Discovery v2
+- `analyzeDocument()` method added: Process a document using the specified collection's settings and return it for realtime use. **Note: This method is only supported on IBM Cloud Pak for Data instances of Discovery.**
+- Interface `QueryResponsePassage` added
+- Interface `QueryResponse` property `passages` added: Passages returned by Discovery.
+
+#### Language Translator
+- `translateDocument()` now supports these subtitle/caption formats: `text/sbv`, `text/srt`, and `text/vtt`
+
+#### Natural Language Understanding
+- Interface `FeaturesResultsMetadata` added: Webpage metadata, such as the author and the title of the page.
+
+##### Speech to Text
+- `recognize()` and `createJob()` now support Canadian French broadband and narrowband models
+- `createLanguageModel()` and `createAcousticModel()` now supports Australian English broadband and narrowband models
+- `addGrammar()` parameter `grammarFile` now supports `ReadableStream` and `Buffer`
+
+#### Visual Recognition v4
+- `createCollection()` parameter `trainingStatus` added: Training status information for the collection.
+- `updateCollection()` parameter `trainingStatus` added: Training status information for the collection.
+- Interface `CollectionTrainingStatus` added: Training status information for the collection.
+- Interface `ObjectDetailLocation` added: Defines the location of the bounding box around the object.
```

Comment:
I'm not going to look at all of these in detail. The format looks good, just make sure that you aren't including anything from the previous release on accident